src/lib/theme/deuteranomaly.js

```diff
@@ -0,0 +1,29 @@
+import createTheme from "/lib/theme/createTheme";
+import colors from "/lib/colors";
+
+const theme = (type = "light") =>
+  createTheme({
+    palette: {
+      type,
+
+      critical: { main: colors.lime[500] },
```

Comment:
I _think_ these need light and dark variants as well

Reply:
light is never used on any of these, so I've removed them. We can get a similar effect with opacity if needed. However, I've kept the dark versions because they're sometimes used to get the desired contrast text